lib/src/config.rs

```diff
@@ -223,4 +247,5 @@ pub struct RepoEncryption {
 pub struct Repo {
     pub version: u32,
     pub encryption: Option<RepoEncryption>,
+    pub chunking: Option<RepoChunking>,
```

Comment:
I don't think `Option` is necessary here. I guess there can't be no chunking settings . It can be missing from the config but https://serde.rs/attr-default.html should take care of it.

I guess similarly `encryption` might not use `Option`.  There can be no encryption, but it should be written as

```
[encryption]
type = "none"
```
or something like it.

So I guess `RepoEncryptoin` should be renamed to `Encryption` and made into:

```
enum RepoEncryption {
     None,
     Curve25519Blake2BSalsa20Poly1305,
}
```

But I guess it's a story for another PR.